go.mod

```diff
@@ -3,11 +3,11 @@ module github.com/scaleway/packer-plugin-scaleway
 go 1.18
 
 require (
-	github.com/hashicorp/hcl/v2 v2.16.2
+	github.com/hashicorp/hcl/v2 v2.14.1
```

Comment:
Why are we downgrading here? I would strongly prefer to raise issue to upstream and wait for a fix to happen.

Reply:
https://github.com/hashicorp/packer-plugin-sdk/issues/135